fixtures/prolights/smartbat.json

```diff
@@ -0,0 +1,270 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "SMARTBAT",
+  "categories": ["Color Changer"],
+  "meta": {
+    "authors": ["Matteo Piccina"],
+    "createDate": "2019-12-23",
+    "lastModifyDate": "2019-12-23"
+  },
+  "links": {
+    "manual": [
+      "https://www.musiclights.it/download.php?id=3844"
+    ],
+    "productPage": [
+      "https://www.musiclights.it/product/SMARTBAT"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=2A7gp6_6tMQ"
+    ]
+  },
+  "physical": {
+    "dimensions": [137.8, 184, 145],
+    "weight": 3.4,
+    "power": 30,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "4× 8W RGBW LED",
+      "lumens": 1127
```

Comment:
Where does this number come from?

Reply:
Found it on product page under “SORGENTE LUMINOSA” -> “Flusso luminoso”

Reply:
Ah, you're right. The manual doesn't mention that (and it has, for some reason, sightly different numbers for physical data than the product page).